control-plane/.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
We should remove this file after https://github.com/google/privacy-sandbox-aggregation-service/pull/256 is merged.